packages/rich-text/src/plugins/Paragraph/Paragraph.tsx

```diff
@@ -10,6 +10,7 @@ const styles = {
   [BLOCKS.PARAGRAPH]: css`
     line-height: ${tokens.lineHeightDefault};
     margin-bottom: 1.5em;
+    direction: inherit;
```

Comment:
All these `inherit`s are here to force direction in Slate. AFAIK, there is no way to disable Slate's internal direction detection